dbt/models/ccao/schema.yml

```diff
@@ -14,6 +14,16 @@ sources:
         tags:
           - type_ic
 
+      - name: corner_lot
+        description: '{{ doc("table_corner_lot") }}'
+        columns:
+          - name: pin10
+            description: '{{ doc("shared_column_pin10") }}'
+          - name: is_corner_lot
+            description: Corner lot indicator.
```

Comment:
**suggestion:** Let's add a `shared_column` definition for the corner lot indicator.